### PR TITLE
Scrape more of _cluster/health

### PR DESCRIPTION
**Pull request by mburke5678**
https://jira.coreos.com/browse/OSDOCS-508

### CODE REVIEW

**Comment by mburke5678** [comment]
@QiaolingTang Please take a look.

File: modules/efk-logging-elasticsearch-status-viewing.adoc [hunk 1]
```diff
@@ -45,18 +45,60 @@ $ oc get Elasticsearch elasticsearch -n openshift-logging -o yaml
...
-  clusterHealth: green <2>
-  conditions: <3>
-      .....
-  nodes:  <4>
-      .....
-  pods: <5>
-      .....
-  shardAllocationEnabled: "All" <6>
+status:  <1>
+  clusterHealth: <2>
```
Comment (QiaolingTang) [inline modules/efk-logging-elasticsearch-status-viewing.adoc]: We have some little changes here:
```
status:
  cluster:
    activePrimaryShards: 30
    activeShards: 60
    initializingShards: 0
    numDataNodes: 3
    numNodes: 3
    pendingTasks: 0
    relocatingShards: 0
    status: green
    unassignedShards: 0
  clusterHealth: ""
  conditions: []
  nodes:
  - deploymentName: elasticsearch-cdm-zjf34ved-1
    upgradeStatus: {}
  - deploymentName: elasticsearch-cdm-zjf34ved-2
    upgradeStatus: {}
  - deploymentName: elasticsearch-cdm-zjf34ved-3
    upgradeStatus: {}
  pods:
    client:
      failed: []
      notReady: []
      ready:
      - elasticsearch-cdm-zjf34ved-1-6d7fbf844f-sn422
      - elasticsearch-cdm-zjf34ved-2-dfbd988bc-qkzjz
      - elasticsearch-cdm-zjf34ved-3-c8f566f7c-t7zkt
    data:
      failed: []
      notReady: []
      ready:
      - elasticsearch-cdm-zjf34ved-1-6d7fbf844f-sn422
      - elasticsearch-cdm-zjf34ved-2-dfbd988bc-qkzjz
      - elasticsearch-cdm-zjf34ved-3-c8f566f7c-t7zkt
    master:
      failed: []
      notReady: []
      ready:
      - elasticsearch-cdm-zjf34ved-1-6d7fbf844f-sn422
      - elasticsearch-cdm-zjf34ved-2-dfbd988bc-qkzjz
      - elasticsearch-cdm-zjf34ved-3-c8f566f7c-t7zkt
  shardAllocationEnabled: all

```

**Comment by mburke5678** [comment]
@QiaolingTang Thank you very much for the output. How does it look?

**Comment by QiaolingTang** [comment]
These changes LGTM, thanks.

**Pull request merged**